packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1363,7 +1365,7 @@ export interface CustomCallControlButtonCallbackArgs {
 }
 
 // @beta
-export type CustomCallControlButtonPlacement = 'first' | 'last' | 'afterCameraButton' | 'afterEndCallButton' | 'afterMicrophoneButton' | 'afterDevicesButton' | 'afterParticipantsButton' | 'afterScreenShareButton';
+export type CustomCallControlButtonPlacement = 'primary' | 'overflow' | 'secondary';
```

Comment:
`CallComposite` doesn't have a secondary at the moment, will using secondary do nothing or do we plan to allow call composite to have a secondary section (that is empty currently?)
Otherwise we should have two types: one for callcomposite and one for callwithchatcomposite

Reply:
My thinking was to provide CallComposite with a secondary (but leave empty/do nothing for now), which readies us the with the option to add a secondary section in the future for CallComposite.

I am fine with splitting it into two types, but thought it would scale better in this case.

Reply:
I wonder if there is value in exposing the secondary area now for the calling composite? allow Contoso to place their own buttons there with their own functionality even though we might not have anything there?

Reply:
I am open to having separate areas for Calling and CallWithChat. However, it would involve more maintenance and would require adding separate custom button files for the two.

Reply:
I separated the implementation between how Calling and CallWithChat custom buttons are injected.